config/e2e/monitoring.yaml

```diff
@@ -227,7 +227,7 @@ spec:
       meta:
         package:
           name: system
-          version: 9.0.0
+          version: 9.1.0
```

Comment:
I have to check this one, IIUC Agent packages have a different lifecycle.

Reply:
Last version of package system seems to be `2.5.1`, not sure why we have the stack version here. I'm wondering it is not a bug in `update-stack-version.sh`.

<img width="616" height="675" alt="image" src="https://github.com/user-attachments/assets/1df1f482-5aa0-43e4-b0b2-900438182135" />